src/main/java/edu/greenblitz/tobyDetermined/subsystems/DriveTrain/IO/SimModuleIO.java

```diff
@@ -0,0 +1,103 @@
+package edu.greenblitz.tobyDetermined.subsystems.DriveTrain.IO;
+
+import edu.greenblitz.tobyDetermined.RobotMap;
+import edu.greenblitz.tobyDetermined.subsystems.swerve.SwerveChassis;
+import edu.greenblitz.utils.Conversions;
+import edu.greenblitz.utils.motors.GBFalcon;
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.math.system.plant.DCMotor;
+import edu.wpi.first.wpilibj.DriverStation;
+import edu.wpi.first.wpilibj.simulation.DCMotorSim;
+import org.littletonrobotics.junction.Logger;
+
+public class SimModuleIO implements IModuleIO {
+
+    private final DCMotorSim linearMotor;
+    private final DCMotorSim angularMotor;
+    private IModuleIOInputsAutoLogged lastInputs = new IModuleIOInputsAutoLogged();
+    private final SwerveChassis.Module module;
+    private final PIDController angularController = new PIDController(
+            RobotMap.Swerve.SimulationSwerve.angularController.kP,
+            RobotMap.Swerve.SimulationSwerve.angularController.kI,
+            RobotMap.Swerve.SimulationSwerve.angularController.kD
+    );
+
+    private double angularAppliedVoltage, linearAppliedVoltage;
+
+    public SimModuleIO(SwerveChassis.Module module) {
+        this.module = module;
+        System.out.println("[Init] simulation swerve module " + this.module.name());
```

Comment:
Use logger instead of printing

Reply:
the prints are just fro me to know if i initialized the right thing. i'll just delete them